gst-libs/mfx/gstmfxencoder.c

```diff
@@ -1026,6 +1120,7 @@ gst_mfx_encoder_prepare (GstMfxEncoder * encoder)
       return GST_MFX_ENCODER_STATUS_ERROR_OPERATION_FAILED;
 
   gst_mfx_encoder_set_frame_info (encoder);
+  priv->params.ExtParam = priv->extparam_internal;
```

Comment:
May have to find a better place to put this statement...

Reply:
Any suggestions where it should go?

Reply:
last line of gst_mfx_encoder_set_encoding_params () seems to be the best place...